mitodl/hugo-course-publisher
# home page second pass

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #357 
closes #355 
closes #354 
part of #314 
closes #313

#### What's this PR do?

this is

- a second pass on the home screen design
- adding the new top banner to all pages
- fixing several issues pointed out about the current implemention of the home screen, like button heights and so on

#### How should this be manually tested?

check out the branch and look at the relevant designs. The home page should have a responsive design that is close to the mocks.

check out the course pages too - the new site header should be seamlessly integrated there as well, and on the course page as well.


#### Screenshots (if appropriate)

home page:

![localhost_3000_(iPhone 6_7_8) (2)](https://user-images.githubusercontent.com/6207644/99578922-81dfb400-29ab-11eb-877a-74c473cdc771.png)

![localhost_3000_ (1)](https://user-images.githubusercontent.com/6207644/99578495-ed755180-29aa-11eb-9c30-5d9719a8439a.png)

course pages:

![localhost_3000_courses_18-065-matrix-methods-in-data-analysis-signal-processing-and-machine-learning-spring-2018_](https://user-images.githubusercontent.com/6207644/99578975-94f28400-29ab-11eb-9d00-3ac55552d30f.png)

search:
![localhost_3000_search_](https://user-images.githubusercontent.com/6207644/99579043-ab004480-29ab-11eb-8b3f-8cb3e02fa7bf.png)
